### PR TITLE
Move Powershell formatting to match arcade

### PR DESCRIPTION
**Pull request by jaredpar**
Two space indent is wrong but it's the standard that Arcade
chose. Moving our Powershell to be consistent with the standard.

:smile:

### CODE REVIEW

**Comment by jaredpar** [comment]
@dotnet/roslyn-infrastructure for review

File: eng/build-utils.ps1 [hunk 1]
```diff
@@ -17,41 +17,41 @@ $bootstrapConfiguration = if (Test-Path variable:bootstrapConfiguration) { $boot
 $properties = if (Test-Path variable:properties) { $properties } else { @() }
 
 function GetProjectOutputBinary([string]$fileName, [string]$projectName = "", [string]$configuration = $script:configuration, [string]$tfm = "net472", [string]$rid = "", [bool]$published = $false) {
-  $projectName = if ($projectName -ne "") { $projectName } else { [System.IO.Path]::GetFileNameWithoutExtension($fileName) }
-  $publishDir = if ($published) { "publish\" } else { "" }
-  $ridDir = if ($rid -ne "") { "$rid\" } else { "" }
-  return Join-Path $ArtifactsDir "bin\$projectName\$configuration\$tfm\$ridDir$publishDir$fileName"
+ $projectName = if ($projectName -ne "") { $projectName } else { [System.IO.Path]::GetFileNameWithoutExtension($fileName) }
```
Comment (tmat) [inline eng/build-utils.ps1]: This doesn't look right. There is only a single space indent.

Comment (jaredpar) [reply eng/build-utils.ps1]: stupid off by one errors ...

Comment (jaredpar) [reply eng/build-utils.ps1]: Fixed

**Comment by sharwell** [comment]
@jaredpar Can you add this to .editorconfig as well?

**Comment by jaredpar** [comment]
@sharwell sure. Can you point to how tabs are defined?

**Comment by sharwell** [comment]
You'll just want to add a section like the one for JSON:

https://github.com/dotnet/roslyn/blob/7cb5a0c9f1274ebb6b5c072da40cedcd034ffc7b/.editorconfig#L25-L27

**Comment by jaredpar** [comment]
@sharwell, @tmat updated

**Comment by jaredpar** [comment]
One day CI will complete before a conflict gets checked in.

**Pull request merged**